src/modal_functions/volumeData_setup.py

```diff
@@ -0,0 +1,103 @@
+import os
+import shutil 
+import logging
+import yaml
+import zipfile
+from dotenv import load_dotenv
+import boto3
+import gdown
+
+# Logging configuration
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
+class DataManager:
+    def __init__(self, s3_bucket_name, env_file=".env"):
+        # Load environment variables
+        load_dotenv(env_file)
+
+        # AWS credentials setup
+        self.s3_client = boto3.client(
+            's3',
+            aws_access_key_id=os.getenv('AWS_ACCESS_KEY_ID'),
+            aws_secret_access_key=os.getenv('AWS_SECRET_ACCESS_KEY')
+        )
+
+        self.bucket_name = s3_bucket_name
+        self.files = {
```

Comment:
nit: `self.files` is a bit ambiguous, we need to use a more explicit var name which tells us what these files are for, for example `files_config_and_maps`.